changelog/v0.1.1/add-uint64value.yaml

```diff
@@ -0,0 +1,3 @@
+changelog:
+  - type: NON_USER_FACING
```

Comment:
nit: This is ok, though I would argue that since this is an internal library, consumed by other people/teams, this is a FIX, since we are fixing the schema, which was previously incorrectly formatted